spec/features/static_pages_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe "The static pages" do
+RSpec.describe "The static pages", :clean_repo do
```

Comment:
Did something in the PR cause the need for `:clean_repo`?

Reply:
No this has been a randomly failing spec for me. It was failing due to some missing thumbnail in the "recent objects" list due to an object that didn't get created fully elsewhere.

Reply:
(Randomly failing both locally and in CI)

Reply:
Thanks for the clarification.  And nice to see changes that make the tests more stable.